catalyst-gateway/bin/Cargo.toml

```diff
@@ -15,15 +15,15 @@ repository.workspace = true
 workspace = true
 
 [dependencies]
-cardano-chain-follower = { version = "0.0.8", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250708-00" }
-rbac-registration = { version = "0.0.4", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250708-00" }
-catalyst-types = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250708-00" }
-cardano-blockchain-types = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250708-00" }
-catalyst-signed-doc = { version = "0.0.4", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250708-00" }
-c509-certificate = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250708-00" }
+cardano-chain-follower = { version = "0.0.9", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250724-00" }
```

Comment:
```suggestion
cardano-chain-follower = { version = "0.0.9", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "cardano-chain-follower-v0.0.9" }
```

Reply:
Revert the other lines, and only change this library.
I already re-tagged cat-libs with this tag.

Reply:
remade a release and applied it